packages/conf/src/index.test.ts

```diff
@@ -143,34 +142,24 @@ describe("@lingui/conf", function () {
       })
     })
 
-    it("loads TypeScript config", function () {
-      // hide validation warning about missing locales
-      mockConsole(() => {
-        const config = getConfig({
-          configPath: path.resolve(
-            __dirname,
-            path.join("fixtures", "valid", "custom.config.ts")
-          ),
-        })
-        expect(config.locales).toEqual(["pl"])
+    it("loads TypeScript config", () => {
+      const config = getConfig({
+        configPath: path.resolve(
+          __dirname,
+          path.join("fixtures", "valid", "custom.config.ts")
+        ),
       })
+      expect(config.locales).toEqual(["pl"])
     })
 
     describe("fallbackLocales logic", () => {
-      afterEach(() => {
-        mockFs.restore()
-      })
-
       it("if fallbackLocale is defined, we use the default one on fallbackLocales", () => {
-        mockFs({
-          ".linguirc": JSON.stringify({
-            locales: ["en-US"],
-            fallbackLocale: "en",
-          }),
-        })
```

Comment:
mockFs is actually very dangerous because it breaks resolution of files in underlying libraries. typescript-comiconfig is not an exclusion. It didn't work with mockFs, so I removed it in favour of real file in FS.